doc/source/computation.rst

```diff
@@ -578,6 +578,7 @@ they are implemented in pandas such that the following two calls are equivalent:
    df.expanding(min_periods=1).mean()[:5]
 
 These have a similar set of methods to ``.rolling`` methods.
+Please see the :ref:`cumsum, cummax, cummin, cumprod <basics.stats>`
```

Comment:
I wold expect a ``note::`` section here explaining the differnce as I have noted in the issue